docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -101,6 +85,68 @@ First, we create the parent table that contains a `geo_partition` column which i
     );
     ```
 
+1. Create transaction tables for use within each region. (Replace the IP addresses with those of your YB-Master servers.)
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+        create_transaction_table transactions_eu_central_1
+    ```
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+        create_transaction_table transactions_us_west_2
+    ```
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+        create_transaction_table transactions_ap_south_1
+    ```
+
+1. Assign placement for each transaction table. (Replace the IP addresses with those of your YB-Master servers.)
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+        modify_table_placement_info system transactions_eu_central_1 \
+        aws.eu-central-1.eu-central-1a,aws.eu-central-1.eu-central-1b,aws.eu-central-1.eu-central-1c 3
+    ```
+
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+        modify_table_placement_info system transactions_us_west_2 \
+        aws.us-west-2.us-west-2a,aws.us-west-2.us-west-2b,aws.us-west-2.us-west-2c 3
+    ```
+    ```sh
+    ./bin/yb-admin \
+        -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.07:7100 \
+        modify_table_placement_info system transactions_ap_south_1 \
+        aws.ap-south-1.ap-south-1a,aws.ap-south-1.ap-south-1b,aws.ap-south-1.ap-south-1c 3
+    ```
+
+## Step 2. Create table with partitions
+
+First, we create the parent table that contains a `geo_partition` column which is used to create list-based partitions for each geographic region we want to partition data into as shown in the following diagram:
+
+![Row-level geo-partitioning](/images/explore/multi-region-deployments/geo-partitioning-1.png)
+
+1. Create the parent table.
+
+    ```sql
+    CREATE TABLE transactions (
```

Comment:
Now that we are introducing the concept of system transactions table, I think the examples in this page are better off using some other user table name - like 'bank_account_transactions'. I actually got confused with the transactions_ system table and the transactions_ user table while going over this example. I am sure a new reader would run into similar issues.